docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,128 @@
+
+## From Chain-centricity to Application-centricity
+
+If we see Polkadot as a service provider of trustless and resilient computation through cores as
+well as secure interoperability between core-powered applications, the future development of
+Polkadot can be directed towards the following main changes.
+
+A paradigm shift from:
+
+- being a chain-focused ecosystem where each parachain owned an execution core at all times
+  (acquired through fixed parachain auction), which allowed a simple and secure, sharded execution
+  environment
+- to being an application-focused ecosystem where we remove the assumption that each application
+  owns a core, and instead that all cores are a resource to be consumed and used as needed by all
+  applications.
+
+Previously, securing a core was a competitive process through an
+candle auction mechanism. With [coretime](#agile-coretime-allocation), there is no need for
+auctions anymore. Teams can purchase on-demand coretime or reserve bulk coretime as required. This
+greatly decreases the barrier-to-entry for software tinkerers and parachain teams. Coretime will also allow any application to access Polkadot's computation based on their needs maximizing the use of valuable blockspace.
+
+## From Slot Auctions to Coretime Marketplace
+
+The end product of blockchains is [**Blockspace**](./polkadot-v1.md#polkadots-blockspace).
+Applications need to access Polkadot's blockspace, and the entry points to blockspace are the cores.
+Thus, applications will need to reserve some time on cores or **Coretime** to gain the right to
+access Polkadot's secure blockspace and interoperability for a finite period.
+
+Cores must be agile and general: they can change what job they run as easily as a modern CPU. It
+follows that the procurement of those cores must be agile as well.
+
+The auction mechanism is not agile, creates high entry barriers, and is designed for long-running
+single applications (i.e., the original Polkadot vision proposed in the whitepaper).
+
+We depart from the classic lease auctions and propose an agile marketplace for coretime, where
+essentially **coretime becomes a commodity that can be tokenized, sold, and traded**. This setup
+maximizes the agility of Polkadot and lets the market figure out the best solution needed for
+applications to be successful.
+
+Applications can reserve **bulk coretime** and **on-demand coretime** depending on their needs. Bulk
+coretime rental will be a standard rental of coretime through a broker system parachain at a fixed
+price for a fixed period of time. On-demand coretime rental will be available through ongoing sale
+of coretime for immediate use at a spot price. This system lowers the barrier to entry for
+prospective builders.
+
+For example, revenues from coretime sales can be burnt, used to fund the Treasury, or used for a mix
+of those options. The topic is currently under discussion. For more information, see
```

Comment:
I suppose because this is the vision section, we don't mind referring to things that may change, correct?

Reply:
good catch, updated